src/lib/backend.ai-client-esm.ts

```diff
@@ -702,6 +702,8 @@ class Client {
     }
     if (this.isManagerVersionCompatibleWith('24.03.4')) {
       this._features['endpoint-extra-mounts'] = true;
+      this._features['max-pending-session-count'] = true;
+      this._features['max-concurrent-sftp-sessions'] = true;
```

Comment:
In the schema, it says that the max_concurrent_sftp_sessions field was added as of 23.03.3. Please check :)

Reply:
same as above. please refer https://github.com/lablup/backend.ai/issues/3413

Reply:
I added comments for that.